client/deposits/list/index.js

```diff
@@ -70,9 +70,7 @@ const getColumns = ( sortByDate ) => [
 
 export const DepositsList = () => {
 	const { deposits, depositsCount, isLoading } = useDeposits( getQuery() );
-	const { depositsSummary, isLoading: isSummaryLoading } = useDepositsSummary(
-		getQuery()
-	);
+	const { depositsSummary } = useDepositsSummary( getQuery() );
```

Comment:
We should keep `isSummaryLoading` here and use it when calculating `isDespositsSummaryDataLoaded`, mostly to clarify that those data are coming from a data store.

Reply:
Yeah it makes sense, better for code context and readability, added back in https://github.com/Automattic/woocommerce-payments/pull/1788/commits/5078ef80d3d3a2f1aed306029ebcaae5f63db474